public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html lang="ko">
```
타이틀도 어플리케이션에 맞게 수정하면 더 좋을 것 같네요